tools/build/libwebrtc/mrwebrtc.ps1

```diff
@@ -59,6 +59,30 @@ function Initialize-BuildEnvironment {
     $env:GYP_MSVS_VERSION = "2019"
 }
 
+# Post-checkout clean-up
+function Clear-PostCheckout([string]$SourceFolder) {
+    # Remove tools/ (except tools/clang/ and tools/protoc_wrapper/)
+    $toolsFolder = Join-Path $SourceFolder "tools" -Resolve
+    Rename-Item -Path $toolsFolder -NewName "_tools"
+    $oldToolsFolder = Join-Path $SourceFolder "_tools" -Resolve
```

Comment:
Does "-resolve" throw  if the path doesn't exist?

Reply:
Yes. And that's OK here because `tools/` always exists, and must be renamed successfully, so `_tools` also always exists.